README.md

```diff
@@ -1 +1,201 @@
-# -portfolio-conference-page-v2
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>HTML/CSS & JavaScript capstone project - Conference page</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [ Video Link](#video-link)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone 1: Conference page] <a name="Capstone 1: Conference page"></a>
+
+**[Capstone]** This is my capstone of module 1.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML, CSS, GIT, Javascript, Github
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Index Page]**
+- **[About Page]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Deploy Page Link](https://alex1779.github.io/-portfolio-conference-page-v2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- VIDEO LINK -->
+
+## 🚀 Video Link <a name="video-link"></a>
+
+- [Video Link Link](https://drive.google.com/file/d/1De8zO9Lmbap_2VdQrHsc8HP5eDfQMPj8/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ **OPTIONAL** ] As this detail was not mentioned by the previous reviewer, kindly make sure your video contains a highlight about a piece of the code you finds interesting or you are proud of making. You can redo your video to add that and still make sure it doesn't exceed the max length of 5 min.